### ISSUE TITLE
input.json

### ISSUE THREAD

**Issue by Nerolation** [nalinbhardwaj/stealthdrop#3]
Is the input.json file the right one for a merkle tree with 20 levels? Seems like there are not enough hashes in the list.

It would be great if someone could tell me how i can create my own local merkle tree to test everything!

Thanks in advance!

**Comment by Divide-By-0** [comment]
That's true, we are hoc upgraded that constant later. You can simply add more values to path indices and path elements and update the merkle root to reach 20!
